plugin/src/main/java/com/h3xstream/findsecbugs/crypto/WeakTLSDetector.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Find Security Bugs
+ * Copyright (c) Philippe Arteau, All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3.0 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library.
+ */
+package com.h3xstream.findsecbugs.crypto;
+
+import edu.umd.cs.findbugs.BugInstance;
+import edu.umd.cs.findbugs.BugReporter;
+import edu.umd.cs.findbugs.OpcodeStack;
+import edu.umd.cs.findbugs.Priorities;
+import edu.umd.cs.findbugs.bcel.OpcodeStackDetector;
+import org.apache.bcel.Constants;
+
+public class WeakTLSDetector extends OpcodeStackDetector {
```

Comment:
FindSecBugs covers a lot of potential dependencies. The dependencies could also be from different versions. Instead of adding them as test dependencies.
We create stub class in this projet: https://github.com/find-sec-bugs/find-sec-bugs/tree/master/plugin-deps/src/main/java

You don't need to recreate all the methods .. Just the minimum need for compilation.